Release 328.0.0 (#5482)
MetaMask/core: packages/multichain-transactions-controller/CHANGELOG.md

## [Unreleased]

> ## [0.8.0]

Reviewer: This could have a been just a patch bump, but this is controller is not yet "stable", so that's probably ok. 😅

### Changed

> - Adds safeguard for keyring being locked or unlocked ([#5473](https://github.com/MetaMask/core/pull/5473))

Reviewer: ```suggestion
- Check if `KeyringController` is unlocked before processing account events in `MultichainTransactionsController` ([#5473](https://github.com/MetaMask/core/pull/5473))
  - This is needed since some Snaps might encrypt their state which needs the `KeyringController` to be unlocked.
```

> - Release 327.0.0 ([#5481](https://github.com/MetaMask/core/pull/5481))

Reviewer: Similar comment:

```suggestion
```

>   - This is needed since some Snaps might encrypt their state which needs the `KeyringController` to be unlocked.

Reviewer: ```suggestion
  - This is needed since some Snaps might decrypt their state which needs the `KeyringController` to be unlocked.
```
